src/Button/OutlineButton.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Button from './Button'
+
+const OutlineButton = styled(Button)`
+  background-color: ${props => props.theme.colors.white};
+  color: ${props => props.color || props.theme.colors.blue};
+  border-width: 2px;
+  border-color: ${props => props.color || props.theme.colors.blue};
+
+  &:hover {
+    background-color: ${props =>  props.theme.colors.white};
```

Comment:
I thought the hover state was a dark blue color?

Reply:
We need that yet!

Reply:
Maybe leave out the hover states and refactor after the colors are added then?